runner/process.py

```diff
@@ -332,6 +332,10 @@ def bins_errs(run, num_bins, max_error_locs):
     barlist = plt.bar(range(ERROR_LOCS_HISTOGRAM_BINS * 2), flat_zip(run1_bins, run2_bins), yerr=flat_zip(run1_errs, run2_errs), \
             align="center", log=True, color=["black", "red"], \
             error_kw={"ecolor": "black", "elinewidth": 1, "capthick": 0.5, "capsize": 1})
+
+    run1_bins = run1_errs = None
+    run2_bins = run2_errs = None
```

Comment:
Maybe want to `gc.collect()` here as well?

Reply:
It turns out that for things that stay entirely within the Python file, `collect()` doesn't really help. So I think we're OK without here.